package.json

```
@@ -2,5 +2,25 @@
   "devDependencies": {
     "@types/node": "^22.5.0",
     "ts-node": "^10.9.2"
-  }
+  },
+  "name": "tsrun",
+  "version": "1.0.0",
+  "description": "TSRun is a wrapper script that guesses command fixes using the Gemini API.",
+  "main": "run.ts",
+  "dependencies": {
+    "typescript": "^5.5.4"
+  },
+  "scripts": {
+    "install": "ln -s $PWD/run.ts /usr/local/bin/t"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Zackareee/TSRun.git"
+  },
+  "author": "github@zackaree.com",
+  "license": "UNLICENSED",
```

```suggestion
  "license": "UNLICENSED",
  "private": true,
```

Consider also adding `"private": true` to prevent accidental publication - https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license